From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Gauthier Provost <gauthier@kobol.io>
Date: Tue, 23 Mar 2021 08:00:10 +0100
Subject: [ARCHEOLOGY] Re-enable network checksum TX offload on rockchip64
 family (#2623)

> X-Git-Archeology: > recovered message: > * Re-enable network checksum TX offload on rockchip64 family
> X-Git-Archeology: > recovered message: > * Adjust gmac PBL setting for rk3399 legacy to fix network issues with MTU 1500
> X-Git-Archeology: > recovered message: > * Adjust gmac PBL setting for rk3328 in rockchip64-legacy to fix network issues with MTU 1500
> X-Git-Archeology: > recovered message: > Co-authored-by: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology: - Revision 5e69351b3dc05829133e893f6efe575ef58c7866: https://github.com/armbian/build/commit/5e69351b3dc05829133e893f6efe575ef58c7866
> X-Git-Archeology:   Date: Tue, 23 Mar 2021 08:00:10 +0100
> X-Git-Archeology:   From: Gauthier Provost <gauthier@kobol.io>
> X-Git-Archeology:   Subject: Re-enable network checksum TX offload on rockchip64 family (#2623)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 0125fc6e2731..e8b26881c506 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -286,6 +286,7 @@
 		resets = <&cru SRST_A_GMAC>;
 		reset-names = "stmmaceth";
 		rockchip,grf = <&grf>;
+		snps,pbl = <0x4>;
 		status = "disabled";
 	};
 
-- 
Armbian

